src/components/RecordsDetail/index.js

```diff
@@ -188,6 +193,18 @@ const RecordsDetail = props => {
       label={t({ comment: 'About minimap message', message: 'about minimap' })}
     /> : null
     }
+        <form className ='refine-search-form' >
+        <input type="hidden" name="category" value={curr_category} />
+        <input type="hidden" name="limit" value="40" />
+        <label for="query" className='refine-search-label'>Refining Search:</label>
```

Comment:
New user-facing text (such as "Refining Search:") should be wrapped in a [`<Trans>` tag](https://lingui.dev/tutorials/react-patterns) to allow for i18n/l10n.